x/deployment/types/events_test.go

```diff
@@ -0,0 +1,187 @@
+package types
+
+import (
+	"fmt"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/pkg/errors"
+
+	"github.com/ovrclk/akash/sdkutil"
+)
+
+var (
+	keyAcc, _ = sdk.AccAddressFromBech32("akash1qtqpdszzakz7ugkey7ka2cmss95z26ygar2mgr")
```

Comment:
good idea.  maybe can have a thing in testutil w/ like 100 hard-coded addresses.

Reply:
Yeah I just pulled that out of local setup.